mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -201,7 +201,7 @@ func stop() error {
 		}
 
 		// create an kubernetes client
-		client, err := utils.CreateKubeClient()
+		client, err := meshkitkube.New([]byte(""))
```

Comment:
also we have a flag for `--kubeconfig` correct? We might wanna honour that value here.

Reply:
We can add it in a new issue. Currently we do not have a `--kubeconfig` flag. But this is a required use case when the user needs to specify a different path for their `kubeconfig` file. This would require changes in all the commands that have `platform: kubernetes` support.